web/plugins/selector_button.js

```diff
@@ -0,0 +1,150 @@
+         * @param [obj.activecolor="#0000ee"] The color of the active button
+		 * @param [obj.inactivecolor="#ffffff"] The color of the inactive button(s)
+		 * @param [obj.buttons=["1",...,"7"]] {String[]} The labels of the buttons
+         * @param [obj.edgelabels=none] {String[]} A 2-element array of labels of the edges on the side of the scale. By default, there is no label.
+         * @param [obj.linebreaks=0] {Number} Number of line breaks (br's) placed after the scale
+         * @param [obj.disable=false] {boolean} If true, the scale will be disabled after used. -- Do I need this?
+         * @param [obj.hide=false] {boolean} If true, the scale will be hidden after used.
+         * @param [obj.runextra=""] {String} A string of JS code to be run after the scale is used, after calling {@link Experigen.trial.continueButtonClick}()
+		 * @returns The html for the scale to be included
+		 */
+        var makeSelectorScale = function(obj) {
+            Experigen.screen().responses++;
+            var newScale = new SelectorScale(Experigen.screen().responses, this.scales.length, obj);
+			this.scales.push(newScale);
+            return newScale.html();
+        }
+
+        /**
+		 * A selector scale.
+		 * @class
+		 * @param response_no The rank of the Selector Scale within responses in the screen
+		 * @param scale_no The rank of the Selector Scale within all Selector Scales in the screen
+		 * @param [obj] A collection of options
+		 * @param [obj.activecolor="#0000ee"] The color of the active button
+		 * @param [obj.inactivecolor="#ffffff"] The color of the inactive button(s)
+		 * @param [obj.buttons=["1",...,"7"]] {String[]} The labels of the buttons
+         * @param [obj.edgelabels=none] {String[]} A 2-element array of labels of the edges on the side of the scale. By default, there is no label.
+         * @param [obj.linebreaks=0] {Number} Number of line breaks (br's) placed after the scale
+         * @param [obj.disable=false] {boolean} If true, the scale will be disabled after used. -- Do I need this?
+         * @param [obj.hide=false] {boolean} If true, the scale will be hidden after used.
+         * @param [obj.runextra=""] {String} A string of JS code to be run after the scale is used, after calling {@link Experigen.trial.continueButtonClick}()
+         */
+        var SelectorScale = function(response_no, scale_no, obj) {
+            var that = this;
+            this.n = response_no;
+            this.scaleN = scale_no;
+            var paramsobj = obj || {};
+            this.activecolor = paramsobj.activecolor || "#0000ee";
+            this.inactivecolor = paramsobj.inactivecolor || "#ffffff";
+            this.buttons = paramsobj.buttons || ["1","2","3","4","5","6","7"];
+            this.edgelabels = paramsobj.edgelabels || [''];
+		    this.linebreaks = paramsobj.linebreaks || 0;
+		    this.buttontype = "button";
+		    this.disable = (paramsobj.disable) ? true  : false;

```

Comment:
javascript trukk: ez roviditheto azzal hogy `!!paramsobj.disable` :D  don't change it tho